go.mod

```diff
@@ -168,3 +168,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-libs => github.com/ycombinator/elastic-agent-libs v0.0.0-20230511231513-e7929043da88
```

Comment:
Must be removed once https://github.com/elastic/elastic-agent-libs/pull/123 is merged.